Makefile

```diff
@@ -105,4 +105,7 @@ format-in-docker:
 
```

Comment:
Check the https://github.com/onomyprotocol/onomy/runs/4066622412?check_suite_focus=true the CI is red